src/utilities/formatDate.ts

```diff
@@ -42,3 +42,7 @@ export const displayShortDate = (
   }
   return `${dateValue}`;
 };
+
+export const requestDate = (value: Date): string => {
+  return value && value.toISOString().substring(0, 10);
```

Comment:
since `value` is declared as a `Date` there is no need to do `value &&`. we cannot pass `undefined` or `null`. right?

Reply:
yeap thats my fault 
im tottaly agree with u